content/en/apps/features/supervision/_index.md

```diff
@@ -46,3 +46,12 @@ The CHT can be configured to create [Tasks]({{< relref "apps/features/tasks" >}}
 ### Deployment Case Study
 [Muso](https://www.musohealth.org), a leading community health organization and major contributor to the CHT, has implemented ”360º supervision”, achieving some of the lowest child mortality rates in sub-Saharan Africa. This model provides dedicated mentorship and supportive supervision to CHWs tailored to each CHW's particular strengths and challenges. A key theme of our [human-centered approach](https://medicmobile.org/design) was the idea of using data to improve one-to-one supervision, rather than using analytics to replace Supervisors. Read more about findings from a recent [randomized controlled trial](https://medicmobile.org/blog/new-study-precision-supervision-and-personalized-feedback-dashboards-improve-chw-performance-in-mali).
 {{% /pageinfo %}}
+
+## Supervisor CHW create
```

Comment:
I don't love this title but I also don't have any great suggestions.... I think the part I don't like is "Create" because it feels/sounds more like a technical description than a functional/human centered description. Just in case it might inspire others, here are some more personal meaningful options... "Provision new users", "Add new users", "Add CHW users", "Set up new CHWs", etc...?

Reply:
@michaelkohn would "Supervisor add new CHW users" work?

Reply:
@esthermmoturi ... or even "Supervisor add users" (just thinking of length and it wrapping text on the right nav if it is so long), but whatever you think is best I'm fine with!